lib/spack/spack/bootstrap.py

```diff
@@ -791,14 +791,40 @@ def patchelf_root_spec():
     # TODO: patchelf is restricted to v0.13 since earlier versions have
     # TODO: bugs that we don't to deal with, while v0.14 requires a C++17
     # TODO: which may not be available on all platforms.
-    return _root_spec("patchelf@0.13.1:0.13.99")
+    return _root_spec("patchelf@0.13.1:")
+
+
+def verify_patchelf(patchelf):
+    """Older patchelf versions can produce broken binaries in many, many
+    different ways, we should guard against that."""
```

Comment:
(minor) Can you add an Args: entry, specifying that this expects an `Executable` object?